src/kinetics/KineticsFactory.cpp

```diff
@@ -94,6 +96,7 @@ unique_ptr<Kinetics> newKinetics(std::vector<ThermoPhase*>& phases,
 
 void addReactions(Kinetics& kin, const AnyMap& phaseNode, const AnyMap& rootNode)
 {
+    fmt::memory_buffer err_Plog_reactions;
```

Comment:
I think it would be better if this were defined closer to the first place its used.